app/views/dogs/_dog.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class='shadow-2xl w-[300px] flex-grow mx-2 my-2 p-6 bg-white'>
```

Comment:
Could we rename this partial to something like `_dog-card.html.erb` (I have paranoia we might have something else name `dog` elsewhere.  This might be a good time to name space it now)

Reply:
sure!

Reply:
@freestylebit updated :)